### ISSUE TITLE
[FEATURE] Plugins ideas

### ISSUE THREAD

**Issue by Ai2473** [3arthh4ckDevelopment/3arthh4ck-Plugins#1]
## Make:
- Notebot: downgrade [this](https://github.com/Nat1anWasTaken/Notebot) or use [this one](https://github.com/Victormeriqui/Notebot) (i would like the new one tho)
- [Schematica](https://github.com/Lunatrius/Schematica) (use the code in the pull requests?)
- Seed overlay (couldn't find a link for this)
- Sk1erLLC [patcher](https://github.com/Sk1erLLC/Patcher)
- Sk1erLLC [OldAnimations](https://github.com/Sk1erLLC/OldAnimations)

## Update:
- Update the [Baritone plugin](https://github.com/3arthqu4ke/3arthh4ck/tree/plugins/baritone)
- Update the [ViaForge plugin](https://github.com/3arthqu4ke/3arthh4ck/blob/fc204c458face499813f0394d06c561bdf5a798d/misc/3arthh4ck-ViaVersionPlugin.zip)

Also it would be nice to completely reverse the [AccountSwitcher plugin](https://github.com/colorblindness/3arthh4ck/releases/tag/latest)

**Comment by NCPUser** [comment]
OldAnimations is good add on client

**Comment by Thecoax** [comment]
Add bebeli555 elytrabot, and also autodupes as autoframedupe and autosalc1 dupe it would be good ideas

**Comment by AGENTISNUM1** [comment]
Yo i liked the old PhobosMainMenu plugin
but it doesnt work anymore :(
any chance you could make that?
